tests/subworkflows/icgc-argo-workflows/merg_sort_dup/test.yml

```diff
@@ -0,0 +1,13 @@
+- name: "merg_sort_dup"
+  command: nextflow run main.nf -entry test_merg_sort_dup --tools 'markdup' -profile docker -c ../../../config/nextflow.config
```

Comment:
Also default should match other `test.yml`
https://github.com/icgc-argo-workflows/argo-modules/blob/3d0992ba3996eef2066f961304d03c1ed69d2174/tests/subworkflows/icgc-argo-workflows/song_score_upload/test.yml#L3
https://github.com/icgc-argo-workflows/argo-modules/blob/3d0992ba3996eef2066f961304d03c1ed69d2174/tests/modules/icgc-argo-workflows/checkinput/test.yml#L4